src/context/SuggestionModal.js

```diff
@@ -98,26 +98,26 @@ export default function SuggestionModal({ schema, user, handleClose }) {
 
 	return (
 		<div>
-			<h1>Suggestion for {schema}</h1>
 			<DialogContent>
-				<DialogContentText>
+				<h1>Suggestion for {schema}</h1>
```

Comment:
Can you add in the string formatting that you mentioned here? https://github.com/mikhael28/paretOS/issues/65
To make it capitalized, etc?